bika/lims/browser/stickers.py

```diff
@@ -52,9 +52,17 @@ class Sticker(BrowserView):
             -- other_worksheet_stickers_...
     """
     template = ViewPageTemplateFile("templates/stickers_preview.pt")
-    item_index = 0
-    current_item = None
-    rendered_items = []
+
+    def __init__(self, context, request):
+        self.item_index = 0
+        self.current_item = None
+        self.rendered_items = []
+        self.copies_count = None
+
+        self.context = context
+        self.request = request
+
+        super(Sticker, self).__init__(context, request)
```

Comment:
Better to call the constructor from the super class at the very beginning, rather than at the end of the `__init__` function